terraform/modules/gcp/gke/main.tf

```diff
@@ -1,148 +1,66 @@
 resource "google_container_cluster" "gke_cluster" {
-  project  = var.project_id
   name     = var.cluster_name
-  
-  location = var.region
-  network            = var.network
-  subnetwork         = var.subnetwork
-  deletion_protection = var.cluster_deletion_protection
-  
-  dynamic "master_authorized_networks_config" {
-    for_each = var.master_authorized_networks_config
-    content {
-      dynamic "cidr_blocks" {
-        for_each = master_authorized_networks_config.value.cidr_blocks
-        content {
-          cidr_block   = lookup(cidr_blocks.value, "cidr_block", "")
-          display_name = lookup(cidr_blocks.value, "display_name", "")
-        }
-      }
-    }
-  }
-
-  addons_config {
-    http_load_balancing {
-      disabled = ! var.http_load_balancing
-    }
-
-    horizontal_pod_autoscaling {
-      disabled = ! var.horizontal_pod_autoscaling
-    }
+  location = var.gke_cluster_location
```

Comment:
Shouldn't we use something like https://github.com/terraform-google-modules/terraform-google-kubernetes-engine ? Like we do on AWS?

Reply:
?